src/library/LocalLoopSaturationDirectActingMax.py

```diff
@@ -0,0 +1,64 @@
+"""
+## Local Loop Performance Verification - Direct Acting Loop Actuator Maximum Saturation
+
+### Description
+
+This verification checks that a direct acting control loop would saturate its actuator to maximum when the error is consistently above the set point.
+
+### Verification logic
+
+If the sensed data values are consistently above its set point, and after a default of 1 hour, the control command is still not saturated to maximum, then the verification fails; Otherwise, it passes.
+
+### Data requirements
+
+- feedback_sensor: feedback sensor reading of the subject to be controlled towards a set point
+- set_point: set point value
+- cmd : control command
+- cmd_max: control command range maximum value
+
+"""
+
+import pandas as pd
+from checklib import RuleCheckBase
+
+
+class LocalLoopSaturationDirectActingMax(RuleCheckBase):
+    points = ["feedback_sensor", "set_point", "cmd", "cmd_max"]
+
+    def saturation_flag(self, t):
+        if 0 <= t["cmd_max"] - t["cmd"] <= 0.01:
```

Comment:
Should the `0.01` be a "user-input" with a default value? Similarly to how we handle tolerances.

Reply:
Is there any circumstances where the command might be negative for some reason? In other words should we use `abs(t["cmd_max"] - t["cmd"])`?

Reply:
for 1), my intention is to require as little user inputs / config as possible, so that 0.01 is used. We can certainly change it to a parameter later;
for 2), if it is negative, I would prefer to fail the verification or raise an error. For now, if it is negative, then this would be considered "not saturated", so it has a larger chance of reporting fail than pass.